### ISSUE TITLE
Bug: The NodePackageInstallTask does not appear to have completed successfully or we ran into a race condition. Please run the `ng add @angular/fire` command again.

### ISSUE THREAD

**Issue by edocbuhtig** [angular/angularfire#2878]
### Version info

**Angular:**
12.1.2

**Firebase:**
^7.0 || ^8.0

**AngularFire:**
6.1.5

**Other :**
firebase-tools: ^8.0.0 || ^9.0.0

### How to reproduce these conditions

**Steps to set up and reproduce**

Create a new Angular app using:

```
ng new demoapp
```
Then add firebase app using below:

```
ng add @angular/fire
```

Output you get as below:

```
Using package manager: npm
✔ Found compatible package version: @angular/fire@6.1.5.
✔ Package information loaded.
 
The package @angular/fire@6.1.5 will be installed and executed.
Would you like to proceed? Yes
✔ Package successfully installed.
UPDATE package.json (1443 bytes)
✔ Packages installed successfully.
The NodePackageInstallTask does not appear to have completed successfully or we ran into a race condition. Please run the `ng add @angular/fire` command again.
```

### Expected behavior

The firebase module must install without any errors

### Actual behavior

```
Using package manager: npm
✔ Found compatible package version: @angular/fire@6.1.5.
✔ Package information loaded.
 
The package @angular/fire@6.1.5 will be installed and executed.
Would you like to proceed? Yes
✔ Package successfully installed.
UPDATE package.json (1443 bytes)
✔ Packages installed successfully.
The NodePackageInstallTask does not appear to have completed successfully or we ran into a race condition. Please run the `ng add @angular/fire` command again.
```

**Comment by icecoconut** [comment]
I also got this issues.

**Comment by jamesdaniels** [comment]
Yes, I added this error message. Run the ng add command for a second time, there appears to be a bug with the angular schematic / race-condition I can't track down. I don't think it's isolated to AngularFire but haven't managed to find a work around.

**Comment by ToledoNicola** [comment]
same problem with angular 12.2.2
```
The package @angular/fire@6.1.5 will be installed and executed.
Would you like to proceed? Yes
✔ Package successfully installed.
UPDATE package.json (1376 bytes)
✔ Packages installed successfully.
The NodePackageInstallTask does not appear to have completed successfully or we ran into a race condition. Please run the `ng add @angular/fire` command again.

**Comment by edocbuhtig** [comment]
@ToledoNicola Solution: just re run ``ng add @angular/fire`` and it will install just fine :)

**Comment by jamesdaniels** [comment]
Addressed in 7.1.0-rc.0. I was having a really hard time taking `firebase-tools` as a peer, I've decided it's best instead to take it as a global install since it's a very heavy weight toolchain. This is working much better in my testing and the race conditions on `ng add @angular/fire@next` have gone away.
